yb-voyager/cmd/importSchemaYugabyteDB.go

```diff
@@ -85,8 +83,14 @@ func YugabyteDBImportSchema(target *tgtdb.Target, exportDir string) {
 
 		reCreateSchema := regexp.MustCompile(`(?i)CREATE SCHEMA public`)
 		sqlInfoArr := createSqlStrInfoArray(importObjectFilePath, importObjectType)
-		errOccured := 0
 		for _, sqlInfo := range sqlInfoArr {
+			if !(strings.HasPrefix(sqlInfo.stmt, "SET ") || strings.HasPrefix(sqlInfo.stmt, "SELECT ")) {
+				if len(sqlInfo.stmt) < 80 {
+					fmt.Printf("%s\n", sqlInfo.stmt)
+				} else {
+					fmt.Printf("%s ...\n", sqlInfo.stmt[:80])
+				}
+			}
```

Comment:
Because `math.Min()` takes `float64` as argument, using `Min()` would require multiple (total 3) type-casts. That's why went with a simple `if...else` block.